### PR TITLE
#347 Added "inspector" element to Agenda

### PR DESCRIPTION
**Pull request by carlosmiranda**
#347: Added "inspector" element to Agenda. Also added "QA Reviewer" column to `agenda.xsl`, added tests.

### CODE REVIEW

File: xml/pmo/agenda/-duplicate-inspector.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<agenda version="0.25" updated="2016-12-29T09:03:21.684Z">
+  <order job="gh:test/test#1">
```
Comment (krzyk) [inline xml/pmo/agenda/-duplicate-inspector.xml]: @carlosmiranda you should have also a `role` here (so this test will fail only for incorrect inspectors)

File: xml/pmo/agenda/-invalid-inspector.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<agenda version="0.25" updated="2016-12-29T09:03:21.684Z">
+  <order job="gh:test/test#1">
```
Comment (krzyk) [inline xml/pmo/agenda/-invalid-inspector.xml]: @carlosmiranda you should have also a `role` here (so this test will fail only for incorrect inspectors)

File: xml/pmo/agenda/-invalid-inspector.xml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    <project>A65TGR50F</project>
+    <title>Test Issue</title>
+    <href>https://github.com/test/test</href>
```
Comment (krzyk) [inline xml/pmo/agenda/-invalid-inspector.xml]: @carlosmiranda I don't see `href` element defined for `order` in `agends.xsd`, so I assume it is not needed

File: xml/pmo/agenda/-duplicate-inspector.xml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    <project>A65TGR50F</project>
+    <title>Test Issue</title>
+    <href>https://github.com/test/test</href>
```
Comment (krzyk) [inline xml/pmo/agenda/-duplicate-inspector.xml]: @carlosmiranda I don't see `href` element defined for `order` in `agends.xsd`, so I assume it is not needed

File: xml/pmo/agenda/-duplicate-inspector.xml (hunk 1 above)
Comment (carlosmiranda) [reply xml/pmo/agenda/-duplicate-inspector.xml]: @krzyk I don't understand. Inspector is only restricted to `login` type, so "invalid inspector" here is simply an invalid login string.

**Comment by carlosmiranda** [comment]
@krzyk updated, please check latest code

**Comment by krzyk** [comment]
@carlosmiranda looks good, thanks

**Comment by krzyk** [comment]
@rultor  merge pls

**Comment by krzyk** [comment]
@g4s8 good to merge, but I see that you are not as architect in .rultor.yml, can you fix it? or should it go through usual issue->PR flow?

**Comment by krzyk** [comment]
@yegor256 could you merge this?

**Comment by krzyk** [comment]
@yegor256 ping

**Comment by 0crat** [comment]
@krzyk[/z](https://www.0crat.com/u/krzyk) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by krzyk** [comment]
@g4s8 good to merge (now you are ARC for rultor)

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#350](https://github.com/zerocracy/datum/issues/350) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@krzyk 
According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only two issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by krzyk** [comment]
@ypshenychka actually there were 4 issues, missing `<role>` elements and invalid `<href>` element, the same two issues in the second file, so a total of 4

**Comment by ypshenychka** [comment]
@krzyk These are repating issues. Basically issue is one, but found in two places. Still counts as one.

**Comment by krzyk** [comment]
@ypshenychka does the policy state that those issues have to be unique?

**Comment by ypshenychka** [comment]
@krzyk It doesn't require much effort to see same issue in another place, so I don't count them as two separate issues. Let's ask @yegor256 @g4s8 what do they think of this case.
Anyways I agree that this should be clearly stated in Policy.

**Comment by g4s8** [comment]
@ypshenychka I think @krzyk is right here, because policy doesn't say that issues should be unique, it only says:
> The code reviewer found at least three problems in the code

either unique or not.

@ypshenychka also you can say what do you think about it here: https://github.com/zerocracy/zerocracy.github.io/issues/17

**Comment by ypshenychka** [comment]
@g4s8 Some time ago we agreed on such rule with @yegor256 but it wasn't implicitly stated anywhere. But to my mind it should be documented in policy. I'll finish with this ticket in accordance with current documented rules and let's move discussion to https://github.com/zerocracy/zerocracy.github.io/issues/17.

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +20 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**